src/CMakeLists.txt

```diff
@@ -460,18 +414,13 @@ if(ENABLE_CAMPAIGN_SERVER)
   
   GetSources("campaignd" campaignd_SRC)
 
-  check_library_exists(rt clock_gettime "time.h" HAVE_CLOCK_GETTIME_RT)
-  if(HAVE_CLOCK_GETTIME_RT)
-	  set(LIBRT rt)
-  endif()
-
   add_executable(campaignd ${campaignd_SRC})
+  
   target_link_libraries(campaignd
-	  wesnoth-core
-	  ${server-external-libs}
-	  ${CRYPTO_LIBRARY}
-	  ${Boost_RANDOM_LIBRARY}
-	  ${LIBRT}
+	    wesnoth-core
```

Comment:
While you're here, please fix the indentation. CMakeLists.txt is supposed to be indented with tabs.